vermouth/data/mappings/elnedyn/modifications.elnedyn22p.mapping

```diff
@@ -0,0 +1,129 @@
+[ modification ]
+[ from ]
+universal
+[ to ]
+elnedyn22p
+
+[ from blocks ]
+C-ter
+[ to blocks ]
+C-ter
+
+[ from nodes ]
+N
+HN
+
+[ from edges ]
+HN N
+N CA
+
+[ mapping ]
+CA BB
+C  BB 0
+O  BB 0
+OXT BB 0
+
+
+[modification]
+[ from ]
+universal
+[ to ]
+elnedyn22p
+[ from blocks ]
+N-ter
+[ to blocks ]
+N-ter
+
+[ from nodes ]
+C
+O
+
+[ from edges ]
+C O
+CA C
+
+[ mapping ]
+HN2 BB 0
+HN3 BB 0
+N   BB 0
+CA  BB
+C   BB 0
+O   BB 0
+
+[ modification ]
+[ from ]
+universal
+[ to ]
+elnedyn22p
+
+[ from blocks ]
+COOH-ter
+[ to blocks ]
+COOH-ter
+
+[ from nodes ]
+N
+HN
+
+[ from edges ]
+HN N
+N CA
+
+[ mapping ]
+HN BB 0
+N BB 0
+CA BB
+C  BB 0
+O  BB 0
+HO BB 0
+OXT BB 0
+
+
+[modification]
+[ from ]
+universal
+[ to ]
+elnedyn22p
+[ from blocks ]
+NH2-ter
+[ to blocks ]
+NH2-ter
+
+[ from nodes ]
+C
+O
+
+[ from edges ]
+C O
+CA C
+
+[ mapping ]
+HN2 BB 0
+N   BB 0
+CA  BB
+C   BB 0
+O   BB 0
+
+[modification]
+[ from ]
+universal
+[ to ]
+elnedyn22p
+[ from blocks ]
+ASP0
+[ to blocks ]
+ASP0
```

Comment:
This is currently causing test failures, since there is (apparently) no ASP0 modification in elnedyn22p.

Reply:
yep missed this one apparently it exits for another elnedyn